### ISSUE TITLE
map and loss calculation on validation set

### ISSUE THREAD

**Issue by 123alaa** [AlexeyAB/darknet#2764]
Hi @AlexeyAB,
Is the map and loss calculation applied on validation set by default?

Just i want to be sure.

**Comment by drapado** [comment]
I think the loss is calculated in the train set (train = train.txt in cfg file) and the mAP in the validation set (valid = valid.txt)

**Comment by AlexeyAB** [comment]
@123alaa Hi,

> Is the map and loss calculation applied on validation set by default?

mAP is calculate for `valid=` in your `obj.data` by default for `detector train` and `detector map`.
